app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -136,11 +136,11 @@ class EventDetailsFragment : Fragment() {
 
         rootView.buttonTickets.setOnClickListener {
             val ticketUrl = currentEvent?.ticketUrl
-            if (Uri.parse(ticketUrl).host != getString(R.string.FRONTEND_HOST) &&
-                !ticketUrl.isNullOrEmpty()) {
-                Utils.openUrl(requireContext(), ticketUrl)
-            } else {
-                loadTicketFragment()
```

Comment:
I think you can just turn `  if (Uri.parse(ticketUrl).host != getString(R.string.FRONTEND_HOST) && !ticketUrl.isNullOrEmpty())` to `if (!ticketUrl.isNullOrEmpty() && Uri.parse(ticketUrl).host != getString(R.string.FRONTEND_HOST))`

Reply:
> if (!ticketUrl.isNullOrEmpty() && Uri.parse(ticketUrl).host != getString(R.string.FRONTEND_HOST))

Oh, it's good if that works. Will check :+1:

Reply:
Yes working. Updated :+1: